scrapers/wv/events.py

```diff
@@ -132,17 +128,43 @@ def scrape_meeting_page(self, url):
                 agenda = event.add_agenda_item(
                     row.text_content().strip().replace("\u25a1", "")
                 )
-                for bill in re.findall(self.bill_regex, row.text_content()):
-                    bill_id = re.sub(r"\.\s*", "", bill[0], flags=re.IGNORECASE)
-                    bill_id = re.sub(r"house bill", "HB", bill_id, flags=re.IGNORECASE)
-                    bill_id = re.sub(r"senate bill", "SB", bill_id, flags=re.IGNORECASE)
+
+                # Matches (SJR, HCR, HB, HR, SCR, SB, HJR, SR) + id
+                # Allows for house, senate, joint, or bill to be fully spelled out
+                # Allows for "." after H, S, J, C, and B
+                # Allows for up to two spaces before the id
+                bills = re.findall(
+                    r"((S\.?|Senate|H\.?|House)\s?((J|C|Joint)\.?\s?)?(B\.?|Bill|R\.?)\s?\s?(\d+))",
+                    row.text_content(),
+                    flags=re.IGNORECASE,
+                )
+
+                component_re = re.compile(r"([A-Z]+)\s*(\d+)", flags=re.IGNORECASE)
+                period_and_whitespace_re = re.compile(r"\.\s*", flags=re.IGNORECASE)
+                house_bill_re = re.compile(r"house bill", flags=re.IGNORECASE)
+                senate_bill_re = re.compile(r"senate bill", flags=re.IGNORECASE)
+
+                for bill in bills:
+                    bill_id = re.sub(period_and_whitespace_re, "", bill[0])
+                    bill_id = re.sub(house_bill_re, "HB", bill_id)
+                    bill_id = re.sub(senate_bill_re, "SB", bill_id)
```

Comment:
These three just need to be reformatted in the form of: `bill_id = period_and_whitespace_re.sub("", bill[0])` etc

Reply:
Oops, sorry didn't catch that, fixed now.